src/components/TradeWidget/TokenRow.tsx

```diff
@@ -170,6 +179,15 @@ const TokenRow: React.FC<Props> = ({
     precision: selectedToken.decimals,
   })
 
+  const [visibleForm, showForm] = useState<'deposit' | void>()
+
+  // Checks innerWidth
+  const showResponsive = !!innerWidth && innerWidth < MEDIA.MOBILE_LARGE_PX
+  useNoScroll(!!visibleForm && showResponsive)
+
+  const isDepositFormVisible = visibleForm == 'deposit'
+  // const isWeth = balance.addressMainnet === WETH_ADDRESS_MAINNET
```

Comment:
?

Reply:
??

Reply:
?????????????

Reply:
I think this is tired Leandro saying delete the fucking comment bro :joy:

Reply:
You are good at this :+1: